backend/tournesol/models/poll.py

```diff
@@ -87,10 +86,8 @@ def entity_cls(self):
 
     @property
     def scale_function(self):
-        if self.algorithm == ALGORITHM_MEHESTAN and self.sigmoid_scale is not None:
-            return lambda x: (4 * MEHESTAN_MAX_SCALED_SCORE / TAU) * \
-                             np.arctan(self.sigmoid_scale * x)
-        return lambda x: x
+        # TODO: deprecate sigmoid scale stuff
```

Comment:
Is there more to do than the current change? If not, let's not leave the comment here. If yes let's do it or make the comment more helpful

Reply:
Not much except that the field `sigmoid_scale` should be removed from the model `Poll` and from the database. And I preferred not to include db migrations that would conflict with other ongoing work.